tests/framework/framework.go

```diff
@@ -26,9 +26,21 @@ const (
 
 // KubernetesReporter is the struct that holds the report info.
 type KubernetesReporter struct {
-	FailureCount int
-	artifactsDir string
-	maxFails     int
+	BackupNamespace string
+	FailureCount    int
+	artifactsDir    string
+	maxFails        int
+}
+
+func getBackupNamespaceFromEnv() string {
+	backupNamespace := os.Getenv("KVP_BACKUP_NS")
```

Comment:
Consider using common consts for "KVP_BACKUP_NS" and "velero"

Reply:
done